### ISSUE TITLE
phpcs very slow

### ISSUE THREAD

**Issue by edmcmanwich** [shelljs/shelljs#440]
### Example ShellJS command to reproduce the error:

``` javascript
var shell = require('shelljs');

shell.exec('phpcs --report=json --standard=PSR2 -n ./test/test.php');
console.log(shell.stdout);
```

**Comment by synaptiko** [comment]
@edmcmanwich If you want you can try to use [my branch](https://github.com/synaptiko/shelljs/tree/use-spawn-sync-for-exec-sync) for the test. It should improve the perf.

**Comment by nfischer** [comment]
This will basically be fixed in #866. Merging into #495.
